File: Javascript/PrimeOrNot.js
function primeNumcheck(number){
    for (let i=2;i<number;i++){
        let count = 0
        if (number % i === 0){
            count = count + 1
        }
    if (count>0){
        return `${number} is not a prime`
    }
    else{
        return `${number} is a prime`
    }
    }
}

console.log(primeNumcheck(53))

File: Python/primeOrNot.py
def primeNumCheck(number):
    count = 0
    for i in range(2,number):
        if number % i ==0:
            count = count +1 
    if count > 0:
        return f"{number} is not a prime"
    else:
        return f"{number} is a prime"
    
print(primeNumCheck(22))